tests/event_threads/Makefile

```diff
@@ -0,0 +1,5 @@
+include ../Makefile.tests_common
+
+USEMODULE += event_thread_highest event_thread_medium event_thread_lowest
```

Comment:
Can we define these as configuration values (CFLAGS) that depend on the module "event_thread"? Otherwise it's hard to configure it using Kconfig.

Reply:
how hard?